src/ape/api/providers.py

```diff
@@ -598,6 +598,35 @@ def get_transaction_trace(  # type: ignore[empty-body]
             Iterator(:class:`~ape.type.trace.TraceFrame`): Transaction execution trace.
         """
 
+    @raises_not_implemented
+    def poll_blocks(  # type: ignore[empty-body]
+        self,
+        stop_block: Optional[int] = None,
+        required_confirmations: Optional[int] = None,
+        new_block_timeout: Optional[int] = None,
+    ) -> Iterator[BlockAPI]: # type: ignore[empty-body]
+        """
+        Poll new blocks.
+        **NOTE**: When a chain reorganization occurs, this method logs an error and
+        yields the missed blocks, even if they were previously yielded with different
+        block numbers.
+        **NOTE**: This is a daemon method; it does not terminate unless an exception occurs
+        or a ``stop`` is given.
+        Args:
```

Comment:
you need newlines in this doc for all this to render properly.
new lines before all the little sections.

(i know when you copy+paste docs sometimes, it will get rid of these on accident, that may be what happened here)